build.gradle

```diff
@@ -100,11 +100,8 @@ dependencies {
  * versions of the site
  */
 bower {
-    'jquery' ('3.1.1') {
-        source 'dist/**'
-    }
 
-    /* needed for jenkins.io */
+    /* needed for jenkins.io, jquery from bootstrap */
     'bootstrap' ('4.0.0-alpha.6') {
```

Comment:
jquery is brought in by bootstrap. Theres some way to resolve this with bower and make the dependency explicit but I don't know how to do that at 10pm